demo/package.json

```
@@ -20,7 +20,8 @@
     "quality:test": "exit 0",
     "quality:format": "prettier --write \"./**/*.{js,jsx,ts,tsx,css,scss,md,yaml,yml,json}\"",
     "quality:lint": "run-s quality:lint:*",
-    "quality:lint:ts": "eslint \"src/**/*.ts\"",
+    "quality:lint:eslint": "eslint \"src/**/*.ts\"",
+    "quality:lint:prettier": "prettier --check src/**/*.ts src/*.ts",
```

```suggestion
    "quality:lint:prettier": "prettier --check **/*.{js,jsx,ts,tsx,css,scss,md,yaml,yml,json}",
```
AFAIK `**/*.ts` is including the root folder as well, hence the removal of `src/*.ts`.

Also, it's good to lint other files as well, not only the `ts` files.

---

`src/**/*.ts` does not files in src folder, tested.
Wont just leaving `**/*` also include `node_modules`, `build` and etc?

---

IIRC, if they are in `.prettierignore` they won't be checked (needs testing).

---

done